manifests/init.pp

```diff
@@ -17,6 +17,7 @@
   Optional[String]                                $cron_monthday = '*',
   Optional[String]                                $log_path      = undef,
   Enum['DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL'] $log_level     = 'WARN',
+  Enum['0', 'DAILY', 'WEEKLY', 'MONTHLY']         $log_rotation  = '0',
```

Comment:
Instead of `0` could be make this a string like `NEVER` or `NONE` and translate to zero or just don't bother passing it? We don't want users to have to think about what `0` means.